tests/testthat/test-def_sources.R

```diff
@@ -6,10 +6,14 @@ test_that("def_sources works", {
                           fail_args = list(lat = 43.03, # needs lng
                                            # invalid lng
                                            list(lat = 43.03, lng = -200),
+                                           # invalid lat
+                                           list(lat = 90, lng = -50),
                                            # wrong argument
                                            source_group = 70,
                                            # wrong type
                                            source_id = "test"),
                           warn_args = list(source_id = -604),
                           col_no = 12, check_null = TRUE)
+
+  expect_true(is.data.frame(def_sources(lat = 43.03, lng = -89.4, sf = FALSE)))
```

Comment:
This is already checked in `rmacrostrat_test_checks()`. Was there a specific reason to add it here?

Reply:
In `rmacrostrat_test_checks()`, you don't explicitly check for `sf = FALSE`, i.e. a `data.frame` output. As this output returns less columns, this was needed outside of `rmacrostrat_test_checks()`. This test increases code coverage to 100% - no other reason than this!